dropwizard-demo/src/main/java/com/dep/resource/DropResource.java

```diff
@@ -21,42 +21,40 @@
 @Path("/api/1.0/twitter")
 public class Controller {
 
+    @GET
+    @Path("/getTweets")
+    public static ArrayList<String> Gettweets() throws TwitterException {
```

Comment:
GetTweet doesn't naming following standard
Please change to getTweets

Reply:
Addressed